purchase_manual_delivery/wizard/create_manual_stock_picking.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class CreateManualStockPickingWizard(models.TransientModel):
+    _name = "create.stock.picking.wizard"
+    _description = "Create Manual Stock Picking Wizard"
+
+    def _default_purchase_order(self):
+        if self.env.context["active_model"] == "purchase.order.line":
+            return (
+                self.env["purchase.order.line"]
+                .browse(self.env.context["active_ids"][0])
+                .order_id
+            )
+        return self.env["purchase.order"].browse(self.env.context["active_id"])
+
+    def _default_location_dest_id(self):
+        return self.env["stock.location"].browse(
+            self._default_purchase_order()._get_destination_location()
+        )
+
+    @api.model
+    def default_get(self, fields):
+        res = super().default_get(fields)
+
+        active_model = self.env.context["active_model"]
+        if active_model == "purchase.order.line":
+            po_line_ids = self.env.context["active_ids"] or []
+            purchase_lines = (
+                self.env["purchase.order.line"]
+                .browse(po_line_ids)
+                .filtered(
+                    lambda p: p.product_id.type in ["product", "consu"]
```

Comment:
I think here it could be just:
```suggestion
                    lambda p: p.product_id.type == "consu"
```
as `"product"` is removed in `18.0`

Reply:
Thanks, all suggestions taken!